src/app/static/src/app/components/plots/utils.ts

```diff
@@ -193,7 +193,22 @@ export const formatOutput = function (value: number | string, format: string, sc
     }
 
     if (!format.includes('%') && accuracy) {
-        ans = Math.round(ans / accuracy) * accuracy
+        /**
+         * When accuracy is set to 100 and selected value is less than 200
+         * barchart disarranges YAxis. Code below checks if value is greater
+         * than 500 before apply 100 scale range. Otherwise, chartJS will
+         * automatically use numeric algorithm to calculate scale range. 
+         * However, if accuracy is less than 100, we simply apply scale range
+         * rounding using the given accuracy value.
+         */
+
+        if (accuracy === 100) {
+            if (ans > 500) {
+                ans = Math.round(ans / accuracy) * accuracy
+            }
+        } else {
+            ans = Math.round(ans / accuracy) * accuracy
+        }
```

Comment:
I think I had a different understand from our discussion with Jeff. What he wants is for accuracy of 100
* If value in data < 200 then show to nearest 10
* If value in data < 5 then just show text "<5"

but we should also make this work for other accuracy values. I think logic could be something like,
```
if (value < 5) {
  show <5
} else if (value < accuracy * 2 && accuracy > 1) {
  round to accuracy / 10
} else {
  round to specified accuracy
}
```

Reply:
He suggested we check this with Mary though. He did also say that if this is a faff then just show the raw number.

Reply:
> I think I had a different understand from our discussion with Jeff. What he wants is for accuracy of 100
> 
> * If value in data < 200 then show to nearest 10

I suggested 200 to Jeff because yAxis displayed nicely when I tested but on testing again with different age groups, it disarranged YAxis. applying 100 accuracy when value is greater than 500 fixes issues with YAxis pretty well. I will inform him I updated to 500. 

I think you should try testing with different age groups to see how it displays.

> * If value in data < 5 then just show text "<5"
Only numeric value is applicable here
> 
> but we should also make this work for other accuracy values. I think logic could be something like,
> 
> ```
> if (value < 5) {
>   show <5
> } else if (value < accuracy * 2 && accuracy > 1) {
>   round to accuracy / 10
> } else {
>   round to specified accuracy
> }
> ```

I have allowed ChartJs to calculate the range using min and max value. What I am trying to avoid is exactly your suggestions where we round value based on unknown range. Your suggested solution disarranges YAxis for value for a lot age groups. Also, we can not show <5, only able to return numeric values.

Reply:
I'm coming to this late and may not understand all the issues. I think the problem is that the accuracy value is sometimes being set quite high relative to the highest value in the data (e.g. accuracy 100 when highest value is e.g. 200). However chartJs wants to show about 10 ticks on the chart, and we're asking it to do it with an accuracy to the nearest 100, so it's doing the best it can and rounding what are basically ticks per 10 to the nearest 100. Is that the problem?

It feels like the problem is greater than the scope of this method, which I think is formatting a single value - wouldn't a high accuracy value be ok for low values, as long as there were some high values in the data? (Although the filtering is handled in the barchart component so maybe we do need to get the accuracy right for the lowest possible values too.) Also, dealing with the issue for accuracy == 100 seems very specific. Don't we also support e.g. accuracy == 1000 potentially?

Would it make more sense to have a separate method which checks the appropriateness of the accuracy value across all data values. e.g. something like 'if there are any non-zero values which are  < 10 * accuracy, adjust accuracy to greatest power of 10 which is < 10 * accuracy`.

Reply:
We mostly support accuracy 100, where accuracy is greater or less than 100, CharJs will calculate the scale range and display YAxis nicely. Alternatively, if we do not want chartJS to handle scale range, we will then have to manually calculate for all expected accuracy. 

The point of this ticket is to show YAxis nicely for all data points. I think you should spin the app and play with different formatting.